### ISSUE TITLE
[A11Y] [iOS] Close button in the preview banner does not have an appropriate label.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#409]
**ENVIRONMENT:**
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

**PRE-REQUISITE:**
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

**STEPS TO REPRODUCE:**

1. Navigate to the URL: https://ampstart.com/
2. Navigate to "Templates" link and double tap on it.
3. Navigate to 'VIEW TEMPLATE' in any of the sections and double tap on it.
4. Try to linear navigate to close button in the preview banner and observe what VoiceOver announces.

**ACTUAL RESULTS:**
Close button is labelled as button.

**EXPECTED RESULTS:**
Close button should have an appropriate label assigned.

**ATTACHMENT:**
Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
Fixed in my branch

[Commit Fix](https://github.com/torch2424/ampstart/commit/ce0003bef4689dcedd22ad896826d418532b7a3f)
